lib/src/model/nameable.dart

```diff
@@ -96,9 +96,14 @@ mixin Nameable implements Privacy {
 
 /// Compares [a] with [b] by name.
 int byName(Nameable a, Nameable b) {
+  if (a is Library && b is Library) {
+    return a.displayName.compareTo(b.displayName);
```

Comment:
should we stay consistent and use `compareAsciiLowerCase...` here too?

Reply:
Actually yes, that's a good catch.